hbase-server/src/main/resources/hbase-webapps/static/css/hbase.css

```diff
@@ -54,3 +54,7 @@ table.tablesorter thead tr .headerSortUp {
 table.tablesorter thead tr .headerSortDown {
     background-image: url(desc.gif);
 }
+
+table.nowrap th, td {
```

Comment:
Do you intend to change the style of all `td` elements, or only those under `table.nowrap`? If the latter, I think you need `table.nowrap th, table.nowrap td`.

Reply:
Oh, right! No I was going to add the "nowrap" class only and changing default td style may make other places unhappy.